docs/tezos-basics/introduction_to_cli_and_rpc.md

```diff
@@ -0,0 +1,120 @@
+---
+id: introduction_to_cli_and_rpc
+title: Introduction to cli and rpc
+---
+
+## Tezos RPC (Remote procedure call)
+RPC [1](https://opentezos.com/tezos-basics/introduction_to_cli_and_rpc#reference) is a client-server protocol where the requesting program is the client and the program providing the service is the server. 
+
+The Tezos node provides a JSON/RPC interface to interact with the Tezos Chain. Note that it is an RPC, and it is JSON based, but it does not follow the “JSON-RPC” protocol.
+
+## Tezos-client
+
+Tezos-client is the client for interacting with a tezos node via RPC. Let's take a look at some examples of how to use it? 
+
+The full documentation is [here](https://tezos.gitlab.io/shell/cli-commands.html).
+
+## How to install
+
+* On Mac OS With [Homebrew](https://brew.sh/):
+```
+$ brew install tezos-client
+```
+
+* On Ubuntu with binary:
+```
+$ sudo add-apt-repository ppa:serokell/tezos && sudo apt-get update
+$ sudo apt-get install -y tezos-client
+```
+* on Fedora with binary:
+```
+$ dnf copr enable -y @Serokell/Tezos && dnf update -y
+$ dnf install -y tezos-client
+```
+* From source with with OPAM:
+    Follow this [link](https://tezos.gitlab.io/introduction/howtoget.html#building-from-sources-via-opam)
+
+### Conexion to a network
+The tezos-client needs to be connected to a Tezos node. You can connect to your own tezos node [2](https://opentezos.com/tezos-basics/introduction_to_cli_and_rpc#reference) or you can use a community node on the Tezos mainnet or on a Tezos testnet.
+[Here](https://tezostaquito.io/docs/rpc_nodes/) you can find a list of community nodes on the mainnet or testnet.
+
+If you use a testnet you need to obtain a faucet to interact with it.
+You can create a faucet on testnet [here](https://faucet.tzalpha.net), and download it.
+
+### Account activation
+Activate your account by replacing #accountName and #faucet with an account name and your downloaded faucet path 
+```
+$ tezos-client activate account #accountName with ~/Downloads/#faucet.json
+```
+
+Example: 
+```
+$ tezos-client activate account user1 with ~/Downloads/tz1VH3sHQ5SNby95S9EtPQBqZrhgv2DqjPvy.json
+```
+### Get Balance
+To search the balance of your account you can execute the following command:
+```
+$ tezos-client get balance for #accountName
+```
+
+### get timestamp
+This call is useful to check if the node is synchronized. It returns the actual time is in UTC so it may differ from your local time
```

Comment:
What output should be expected if the node is synchronized or not synchronized ?